sdk/nodejs/runtime/debuggable.ts

```diff
@@ -111,7 +111,8 @@ export function debuggablePromise<T>(p: Promise<T>, ctx: any): Promise<T> {
         .catch((err: any) => {
             localStore.leakCandidates.delete(p);
             err.promise = p;
-            throw err;
+            // Print ctx to stderr which indicates the origin of the rejected promise.
+            console.error(`rejection in: ${ctx}`);
```

Comment:
Feels odd to push this all the way to debuggable promise? We use debuggablePromise is a lot of other places. I'd think it's only in the resource RPC calls (where we don't have any final await for the promise) that this needs to console.errror.

Reply:
Yeah. Some integration tests are failing. Will take a look into making `throwlessDebuggablePromise` (name subject to change) and replace usages of it in `runtime/resource.ts` tomorrow.